uint32 uid
ltm_samples/PersonEntity msg
---
ltm_samples/PersonEntity msg
bool succeeded